src/lib/ThumbnailsSidebar.js

```diff
@@ -311,6 +311,10 @@ class ThumbnailsSidebar {
      * @return {Promise} - promise reolves with the image HTMLElement or null if generation is in progress
      */
     createThumbnailImage(itemIndex) {
```

Comment:
It looks like this method is called as part of a `requestAnimationFrame` callback. Should we track the RAF request and cancel it in `destroy`? That's the only place the cache is destroyed, looks like.